hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OMDBCheckpointServlet.java

```diff
@@ -350,13 +352,13 @@ private boolean includeSnapshotData(HttpServletRequest request) {
     return Boolean.parseBoolean(includeParam);
   }
 
-  private void writeFilesToArchive(Set<Path> copyFiles,
-                                   Map<Path, Path> hardLinkFiles,
-                                   ArchiveOutputStream archiveOutputStream,
-                                   boolean completed)
+  private void writeFilesToArchive(
+      Set<Path> copyFiles,
+      Map<Path, Path> hardLinkFiles,
+      ArchiveOutputStream archiveOutputStream,
+      boolean completed, Path checkpointLocation)
```

Comment:
nit
```suggestion
      boolean completed,
      Path checkpointLocation)
```

Reply:
@smengcl  - Thanks for your review. Taken care.